apps/workplace/src/app/explore/explore.module.ts

```diff
@@ -23,6 +25,8 @@ const ROUTES: Route[] = [
         SharedComponentModule,
         SharedExploreModule,
         FormsModule,
+        MatButtonToggleModule,
+        MatIconModule,
```

Comment:
Use the icon component I made instead of mat icons.

Reply:
I've updated this now.